### PR TITLE
4.8.2 - v4 Migration Cleanup

### PR DESCRIPTION
**Pull request by rejahrehim**
This PR is part of issue #84

- [ ] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Fix linting issues
- Fix dead links

### CODE REVIEW

File: document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.2_Testing_for_Stored_Cross_Site_Scripting_OTG-INPVAL-002.md [hunk 1]
```diff
@@ -1,225 +1,205 @@
...
 
-The input is stored and the XSS payload is executed by the browser when reloading the page. If the input is escaped by the application, testers should test the application for XSS filters. For instance, if the string “SCRIPT” is replaced by a space or by a NULL character then this could be a potential sign of XSS filtering in action. Many techniques exist in order to evade input filters (see [testing for reflected XSS](Testing_for_Reflected_Cross_site_scripting_(OTG-INPVAL-001) "wikilink") chapter). It is strongly recommended that testers refer to [XSS Filter Evasion](XSS_Filter_Evasion_Cheat_Sheet "wikilink") , [RSnake](http://ha.ckers.org/xss.html) and [Mario](https://h4k.in/encoding/) XSS Cheat pages, which provide an extensive list of XSS attacks and filtering bypasses. Refer to the whitepapers and tools section for more detailed information.
+> ![Stored XSS Exxample](images/Stored_xss_example.jpg)\
```
[the remark this answers is not in the document]
Comment (rejahrehim) [reply document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.2_Testing_for_Stored_Cross_Site_Scripting_OTG-INPVAL-002.md]: This section was under "Expected Results" Title. All expected result section will be in notes. Right?

[the remark this answers is not in the document]
Comment (ThunderSon) [reply document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.2_Testing_for_Stored_Cross_Site_Scripting_OTG-INPVAL-002.md]: Right. The images made it a bit awkward. Keep them as such.

File: document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.2_Testing_for_Stored_Cross_Site_Scripting_OTG-INPVAL-002.md [hunk 2]
```diff
@@ -1,225 +1,205 @@
...
 
 When the user loads the page index2.php, the script hook.js is executed by the browser. It is then possible to access cookies, user screenshot, user clipboard, and launch complex XSS attacks.
 
-**Result Expected**
-
-![](RubyBeef.png "RubyBeef.png")
-
-This attack is particularly effective in vulnerable pages that are viewed by many users with different privileges.
+> ![Beef Injection Example](images/RubyBeef.png)\
+> *Figure 4.8.2-2:Beef Injection Example*
```
Comment (kingthorin) [inline document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.2_Testing_for_Stored_Cross_Site_Scripting_OTG-INPVAL-002.md]: This is 4.8.2-3

Comment (rejahrehim) [reply document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.2_Testing_for_Stored_Cross_Site_Scripting_OTG-INPVAL-002.md]: @kingthorin Fixed.

**Pull request merged**